ModernFlyouts/MultilingualResources/ModernFlyouts.nl.xlf

```diff
@@ -8,429 +8,429 @@
           <source>Previous</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">Vorige</target>
+          <target state="final" state-qualifier="tm-suggestion">Vorige</target>
         </trans-unit>
         <trans-unit id="SessionControl.RepeatAll" translate="yes" xml:space="preserve">
           <source>Repeat: all</source>
-          <target state="needs-review-translation" state-qualifier="mt-suggestion">Herhaal: alle</target>
+          <target state="final" state-qualifier="mt-suggestion">Herhaal: alle</target>
         </trans-unit>
         <trans-unit id="SessionControl.RepeatOff" translate="yes" xml:space="preserve">
           <source>Repeat: off</source>
-          <target state="needs-review-translation" state-qualifier="mt-suggestion">Herhaal: uit</target>
+          <target state="final" state-qualifier="mt-suggestion">Herhaal: uit</target>
         </trans-unit>
         <trans-unit id="SessionControl.RepeatOne" translate="yes" xml:space="preserve">
           <source>Repeat: one</source>
-          <target state="needs-review-translation" state-qualifier="mt-suggestion">Herhaal: een</target>
+          <target state="final" state-qualifier="mt-suggestion">Herhaal: een</target>
         </trans-unit>
         <trans-unit id="SessionControl.ShuffleOff" translate="yes" xml:space="preserve">
           <source>Shuffle: off</source>
-          <target state="needs-review-translation" state-qualifier="mt-suggestion">Shuffle: uit</target>
+          <target state="final" state-qualifier="mt-suggestion">Shuffle: uit</target>
         </trans-unit>
         <trans-unit id="SessionControl.ShuffleOn" translate="yes" xml:space="preserve">
           <source>Shuffle: on</source>
-          <target state="needs-review-translation" state-qualifier="mt-suggestion">Shuffle: op</target>
+          <target state="final" state-qualifier="mt-suggestion">Shuffle: aan</target>
         </trans-unit>
         <trans-unit id="SessionControl.Stop" translate="yes" xml:space="preserve">
           <source>Stop</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">Stoppen</target>
+          <target state="final" state-qualifier="tm-suggestion">Stoppen</target>
         </trans-unit>
         <trans-unit id="SessionControl.TimelineInfo" translate="yes" xml:space="preserve">
           <source>Timeline Info</source>
-          <target state="needs-review-translation" state-qualifier="mt-suggestion">Tijdlijngegevens</target>
+          <target state="final" state-qualifier="mt-suggestion">Tijdlijninfo</target>
         </trans-unit>
         <trans-unit id="Settings.Additional" translate="yes" xml:space="preserve">
           <source>Additional</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">Aanvullend</target>
+          <target state="final" state-qualifier="tm-suggestion">Aanvullend</target>
```

Comment:
this can either be 'aanvullend' or 'aanvullende' depending on the sentence/usage.